### ISSUE TITLE
[Issue] `azd init` bug with Azure Table Storage?

### ISSUE THREAD

**Issue by justinyoo** [Azure/azure-dev#2982]
- [x] Make sure you've installed the latest version using [instructions in the wiki](../wiki/install) 👉 I've got v1.4.5

**Output from `azd version`**

```bash
azd version 1.4.5 (commit 6cb723e44eb0697d4d8d09f7694169f5410c7d6f)
```

**Describe the bug**

While provisioning and deploying Aspire-enabled app, I encountered the unsupported resource type issue like this screenshot:

![image](https://github.com/Azure/azure-dev/assets/1538528/6b23da40-e2ab-4824-ab9a-b40ff83b2d5d)

The Aspire-enabled app uses both Azure Queue Storage and Table Storage.

**To Reproduce**

1. Clone my sample app repository: https://github.com/devkimchi/aspire-youtube-summariser
2. Run `azd init`
3. See the error screen like above

**Expected behavior**

`azd init` should be successfully run and generate bootstrap files like `azure.yaml`.

**Environment**
Information on your environment:
    * Language name and version: C# (.NET 8)
    * IDE and version : Visual Studio 2022 Enterprise Preview, v17.9

**Additional context**

N/A

**Comment by vhvb1989** [comment]
azd 1.5.0 provides support for storage blobs.  Adding support for tables should be straight forward.  I will be working on this @savannahostrowski @ellismg

**Comment by justinyoo** [comment]
@vhvb1989 Will this #2998 be released in the next release, say v1.5.1?

**Comment by vhvb1989** [comment]
> @vhvb1989 Will this #2998 be released in the next release, say v1.5.1?

Yes.
you can start using it from the daily build. 
You can install daily from powershell -> `powershell -ex AllSigned -c "Invoke-RestMethod '[https://aka.ms/install-azd.ps1'](https://aka.ms/install-azd.ps1%27) -OutFile 'install-azd.ps1'; ./install-azd.ps1 -Version 'daily'"`

**Comment by VincentH-Net** [comment]
> > @vhvb1989 Will this #2998 be released in the next release, say v1.5.1?
> 
> Yes. you can start using it from the daily build. You can install daily from powershell -> `powershell -ex AllSigned -c "Invoke-RestMethod '[https://aka.ms/install-azd.ps1'](https://aka.ms/install-azd.ps1%27) -OutFile 'install-azd.ps1'; ./install-azd.ps1 -Version 'daily'"`

I tried the daily yesterday (1.6.0 beta) - still have the same error. See context [here](https://github.com/dotnet/aspire/issues/724#issuecomment-1827725800). 

Is there a test / example available that demonstrates this is fixed?

**Comment by vhvb1989** [comment]
Hello @VincentH-Net , I used the repo referenced on this issue to test it [devkimchi/aspire-youtube-summariser](https://github.com/devkimchi/aspire-youtube-summariser)

I checked Linux and Windows, with azd version 1.6.0-beta.1-daily.3273852 (commit f00f37d70ae38a8b53d28650b0398ea4d097ec7d)

Can you add `--debug` at the end of the azd command that is failing and share it?
